docs/pages/build/building-from-github.mdx

```diff
@@ -12,6 +12,23 @@ This guide explains how to trigger builds directly from your GitHub repository u
 
 ## Prerequisites
 
+### Set the `image` field in your eas.json
+
+For the build profiles you want to use with GitHub, you need to specify the image to use for the
+build. This is done in the **eas.json** file in your project's root directory. See [the eas.json reference](/eas/json/#image) to learn more.
+
+```json eas.json
+{
+  // ...
+  "build": {
+    "production": {
+      "ios": {
+        "image": "latest"
+      }
+    }
```

Comment:
```suggestion
    "production": {
     "android": {
        "image": "latest"
      },
      "ios": {
        "image": "latest"
      }
    }
```

Let's also add `android` here because in some cases, this example snippet might bel just copy-pasted and then a question will come arise if the same config required for android too or why it didn't work for android. Specifying both platforms, we try to eliminate the the room for confusion.